.changeset/strong-pets-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+Support `.strict()` on content collection schemas when a custom `slug` is present.
```

Comment:
```suggestion
Adds support for `.strict()` on Content Collection schemas when a custom `slug` is present
```

Reply:
We don’t usually capitalize “content collection”! But I do agree with “Adds support for…” vs “Support…” as per https://contribute.docs.astro.build/guides/new-feature-docs/#changesets

```suggestion
Adds support for `.strict()` on content collection schemas when a custom `slug` is present.
```

Reply:
Ah didn't know that! I'll remember it